src/main/kotlin/com/terraformation/backend/tracking/api/DeliveriesController.kt

```diff
@@ -51,6 +51,7 @@ class DeliveriesController(
   }
 }
 
+// response payload
```

Comment:
Rather than adding comments like this, should we maybe rename these classes to make it clear which ones are for requests vs. responses?

Reply:
As a note, looks like we have about 280 instances of `type <bla> = components['schemas']['SomeType']` and about 210 instances of using `paths`, so switching all over to `paths` might take quite a while. I'll work on converting the ones that directly reference a type with 'zone' in the name for now

Reply:
Actually it looks like none of the types defined in `src/types` have been defined using `paths` since these are supposed to be path-agnostic types :thinking_face: Hmm still thinking how I want to handle this in the FE. Anyway, will merge this one for now.